lib/features/common/service/error_reports/error_reports_service.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter_template/features/common/domain/entity/failures/api_failure.dart';
+import 'package:flutter_template/features/common/domain/entity/operation/failure.dart';
+import 'package:flutter_template/features/common/service/error_reports/i_error_reports_service.dart';
+
+/// {@template error_reports_service.class}
+/// Implementation [IErrorReportsService]
+/// {@endtemplate}
+final class ErrorReportsService implements IErrorReportsService {
```

Comment:
What does 'service' mean in the context of our architecture? Where should it be used?